docs/usage/python.md

```diff
@@ -14,22 +14,25 @@ Renovate supports the following Python package managers:
 
 ## Versioning support
 
-The [PEP440](https://www.python.org/dev/peps/pep-0440/) versioning scheme has been rewritten for JavaScript for the purposes of use in this project is published as [@renovatebot/pep440](https://github.com/renovatebot/pep440).
-It supports both pinned versions as well as ranges.
-Legacy versions (`===` prefix) are ignored.
+We've written a TypeScript version of the [PEP440 specification](https://www.python.org/dev/peps/pep-0440/) so we can use it in Renovate bot.
```

Comment:
Check me if changing the programming language to TypeScript is correct.

Reply:
No, it's actually in JS

Reply:
```suggestion
We've written a JavaScript version of the [PEP440 specification](https://www.python.org/dev/peps/pep-0440/) so we can use it in Renovate bot.
```